### PR TITLE
Bug fix for select2 in embedded panel (task #9973)

### PR DESCRIPTION
**Pull request by eliamariutti**
`select2()` was not initialize correctly in the embedded FieldHandler

### CODE REVIEW

File: src/Template/Element/FieldHandlers/CountryFieldHandler/input.ctp [hunk 1]
```diff
@@ -10,25 +10,6 @@
...
-    [
-        'AdminLTE./bower_components/select2/dist/js/select2.full.min',
-        'Qobo/Utils.select2.init',
-        'RolesCapabilities.switch-target',
-        'RolesCapabilities.permissions',
-    ],
-    ['block' => 'scriptBottom']
-);
-
-$this->Html->css(
```
Comment (softius) [inline src/Template/Element/FieldHandlers/CountryFieldHandler/input.ctp]: @eliamariutti it seems that we have removed 7 assets files and introduced only one in common libs. Are the remaining assets not necessary  or being loaded somewhere else?

Comment (eliamariutti) [reply src/Template/Element/FieldHandlers/CountryFieldHandler/input.ctp]: Yes, all the `FiledHandler` has a `common_js_libs.ctp` with the js and css. 
I think is not necessary to load `RolesCapabilities` js for a country select.

File: src/Template/Element/common_js_libs.ctp [hunk 2]
```diff
@@ -72,6 +72,7 @@ echo $this->Html->script(
         'CsvMigrations.plugin',
         'Qobo/Utils./plugins/tinymce/tinymce.min',
         'CsvMigrations.tinymce.init',
+        'Qobo/Utils.select2.init',
```
Comment (georgeconstantinou) [inline src/Template/Element/common_js_libs.ctp]: @eliamariutti on line 71 we load select2.init file from CsvMigrations plugin, what are the differences between the two?

Comment (eliamariutti) [reply src/Template/Element/common_js_libs.ctp]: @georgeconstantinou From what I have understood in the CsvMigrations plugin, manly we find the Select2 Logic while in the `Utils.select2.init` we just find the `$('.select2').select(...)` to makes everything work

**Pull request merged**